src/data/services/tax.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "service": "Tax Directory (Calculator, Forms, Tables, etc.)",
```

Comment:
service should describe what you can do with the service, for consistency with the other services
e.g. "Calculate Income/VAT Tax and View Tax Requirements and Forms"